### DISCUSSION TITLE
[Enhancement] Logo for Maui

### DISCUSSION THREAD

**Discussion by filipoff2** [dotnet/maui#360] [Ideas]
)## Summary
Win UI have logo.
Logo is required for marketing and sale reasons.

## API Changes
Logo, Surfing net bot is not logo. 
We are still in preview so it can be changed.
 
## Intended Use Case
It require some accent to xamarin.

To new customer it will clear communication that is taking best practices from xamarin.forms not something totally new.

To old customers make them calm that is just rebrand  .

**Comment by filipoff2** [comment]
Surfing .net bot  is more a mascot.

Logo same as Font  have principles how should be done.

Please Ask and talk to graphics designer  what is incorrect with it now.  
Or Google it "principles of making good logo"
Or see some Adobe samples
![unnamed.gif](https://user-images.githubusercontent.com/15687714/109701009-d1c93c00-7b92-11eb-8f31-86f60da1357e.gif)

**Reply by filipoff2** [reply]
Also some ideas an d pros in
https://github.com/dotnet/maui/discussions/340

**Comment by AndyDentFree** [comment]
Please clarify if this will be the default app logo for apps made with Maui, same as the Xamarin logo is at present. Assuming that's the case, it broadens the eyeballs who will see the logo considerably. Every time someone creates a little sample app with Maui it will probably have that logo (most sample apps on GitHub don't bother changing logo).

**Reply by bbenetskyy** [reply]
It's a good point to additional voting when we have some logo selected. 

Right now we have only a generic one, which not represent MAUI and was given like temporary template. 

In my own opinion if we have Evolution from Xamarin to MAUI we should change not only the Logo on Main Page and stickers but also predefined images for apps created from an official template. Because for new developers who just come it may be strange to create MAUI projects with some strange "X" logos 😆
